docs/quick-start-guide.md

```diff
@@ -343,21 +337,23 @@ b. Run the following command to start the CarbonData thrift server.
 ```
 ./bin/spark-submit \
 --class org.apache.carbondata.spark.thriftserver.CarbonThriftServer \
-$SPARK_HOME/carbonlib/$CARBON_ASSEMBLY_JAR <carbon_store_path>
+$SPARK_HOME/carbonlib/$CARBON_ASSEMBLY_JAR <access_key> <secret_key> <endpoint>
```

Comment:
AK SK endpoint is optional, not mandatory.
so remove it from here and other places. Add a note that, to work with s3. 
you can follow below command.
<Add command with ak sk endpoint >

Reply:
ok